### DISCUSSION TITLE
What is the solution for Error 131047 ?

### DISCUSSION THREAD

**Discussion by kvvaradha** [netflie/whatsapp-cloud-api#22] [Q&A] [answered]
I was trying to use the cloud api to send wahtsapp message. but it fails everytime. 

Message failed to send because more than 24 hours have passed since the customer last replied to this number.

Possible solution: Send a business-intiated message using [a message template to respond](https://developers.facebook.com/docs/whatsapp/cloud-api/guides/send-message-templates).

Why am i forced to go with template  while testing with short messages ?

**Comment by aalbarca** [comment, marked as answer]
Are you trying with a testing o production phone number?

**Reply by kvvaradha** [reply]
Thank you for the prompt reply dude. 

i am trying with testing phone number. i found the solution, we have to give a reply from our testing phone number to the whatsapp test account and than we can receive messages from it,

**Comment by kvvaradha** [comment]
I have opened another discussion can you answer me whether we are performing the status read update to whatsapp  ?
